src/Security/Authorization/Core/src/AuthorizationServiceExtensions.cs

```diff
@@ -58,8 +58,8 @@ public static Task<AuthorizationResult> AuthorizeAsync(this IAuthorizationServic
     /// <param name="user">The user to evaluate the policy against.</param>
     /// <param name="policy">The policy to evaluate.</param>
     /// <returns>
-    /// A flag indicating whether policy evaluation has succeeded or failed.
-    /// This value is <c>true</c> when the user fulfills the policy, otherwise <c>false</c>.
+    /// A <see cref="Task{TResult}"/> that contains an <see cref="AuthorizationResult"/> indicating whether policy evaluation has succeeded or failed.
+    /// The result's <see cref="AuthorizationResult.Succeeded"/> property is <c>true</c> when the user fulfills the policy, otherwise <c>false</c>.
```

Comment:
```suggestion
    /// The result's <see cref="AuthorizationResult.Succeeded"/> property is <c>true</c> when the user fulfills the policy; otherwise <c>false</c>.
```